tm2py/core/logging.py

```diff
@@ -0,0 +1,26 @@
+"""Logging module
+
+"""
+
+
+class Logger:
+    """Logger"""
```

Comment:
Be sure and add logging to most of the things you are doing - add lots of good info and debug statements per @lmz preferred verboseness